backend/src/nodes/nodes/image_filter/high_boost_sharpen.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import HbfTypeDropdown, ImageInput
+from ...properties.outputs import ImageOutput
+from ...utils.image_utils import KernelType
+from . import category as ImageFilterCategory
+
+
+@NodeFactory.register("chainner:image:sharpen_hbf")
+class HbfSharpenNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply sharpening to an image using a high boost filter."
+        self.inputs = [
+            ImageInput(),
+            HbfTypeDropdown(),
+        ]
+        self.outputs = [ImageOutput(image_type="Input0")]
+        self.category = ImageFilterCategory
+        self.name = "Sharpen (High Boost)"
+        self.icon = "MdBlurOff"
+        self.sub = "Blur/Sharpen"
+
+    def run(
+        self,
+        img: np.ndarray,
+        kernel_type: int,
+    ) -> np.ndarray:
+        if kernel_type == KernelType.STRONG:
+            # 8-neighbor kernel
+            kernel = np.array([[-1, -1, -1], [-1, 9, -1], [-1, -1, -1]])
```

Comment:
Could we get a sharpen amount here too? The formula for how to generate a kernel from a specific amount can be found [here for NORMAL](https://en.wikipedia.org/wiki/Unsharp_masking#Implementation). The idea is the same for STRONG: just replace the [[0,1,0],[1,1,1],[0,1,0]]/5 kernel in the formula with a [[1,1,1],[1,1,1],[1,1,1]]/9 kernel.

Reply:
That makes a lot more sense than the example I'd found, which weakened the effect at higher values lol.